src/test/java/com/puppycrawl/tools/checkstyle/checks/ClassResolverTest.java

```diff
@@ -73,4 +82,74 @@ public void testMisc() throws ClassNotFoundException {
             // expected
         }
     }
+
+    @Test
+    public void testResolve() throws Exception {
+        final Set<String> imps = Sets.newHashSet();
+        imps.add("java.io.File");
+        imps.add("nothing.will.match.*");
+        imps.add("java.applet.*");
+        imps.add("java.applet.blablabla");
```

Comment:
Can we use some dictionary word here instead of `blablabla`? It doesn't look professional :)